api/command_join_test.go

```diff
@@ -41,18 +41,7 @@ func TestJoinCommands(t *testing.T) {
 
 	c1 := Client.Must(Client.GetChannels("")).Data.(*model.ChannelList)
 
-	if len(c1.Channels) != 6 { // 4 because of town-square, off-topic and direct
-		t.Fatal("didn't join channel")
-	}
-
-	found := false
-	for _, c := range c1.Channels {
-		if c.Name == channel2.Name {
-			found = true
-			break
-		}
-	}
-	if !found {
+	if len(c1.Channels) != 5 { // 5 because of town-square, off-topic, 2 direct channels, and 1 created channel
```

Comment:
I don't see two direct channels. I count town-square, off-topic, channel0, channel2 and the direct channel.